### PR TITLE
Prac 09 feedback

### PR DESCRIPTION
**Pull request by a-rosa**
@JevanChandra
@nora-zyy
@AdriantoHartono
@Zunidzk

### CODE REVIEW

**Review by AdriantoHartono — COMMENTED** [review]
Great code, all the code has been written neatly and can give output as expected 👌

File: prac_09/band.py
Comment (JevanChandra) [inline prac_09/band.py:1]: Great job

File: prac_09/guitar.py
Comment (JevanChandra) [inline prac_09/guitar.py:1]: Nice work

File: prac_09/musician.py
Comment (JevanChandra) [inline prac_09/musician.py:1]: Great code

File: prac_09/my_band.py
Comment (JevanChandra) [inline prac_09/my_band.py:1]: Very nice

File: prac_09/silver_service_taxi.py
Comment (JevanChandra) [inline prac_09/silver_service_taxi.py:1]: Great code

File: prac_09/silver_service_taxi_test.py
Comment (JevanChandra) [inline prac_09/silver_service_taxi_test.py:1]: Nice

File: prac_09/taxi_simulator.py
Comment (JevanChandra) [inline prac_09/taxi_simulator.py:1]: Very long but it works good job

File: prac_09/taxi.py
Comment (JevanChandra) [inline prac_09/taxi.py:1]: Nice one

File: prac_09/taxi_test.py
Comment (JevanChandra) [inline prac_09/taxi_test.py:1]: Great code

File: prac_09/unreliable_car.py
Comment (JevanChandra) [inline prac_09/unreliable_car.py:1]: Nice work

File: prac_09/unreliable_car_test.py
Comment (JevanChandra) [inline prac_09/unreliable_car_test.py:1]: Good job

File: prac_09/taxi_test.py
Comment (nora-zyy) [reply prac_09/taxi_test.py:1]: Clear thinking and easy to understand.

File: prac_09/guitar.py
Comment (nora-zyy) [reply prac_09/guitar.py:1]: It was clearly completed as required.

**Pull request merged**